Bottom: a615be418da44308c2bf5e2cae3ec4ebdfea9ab0
Top:    42de43f6d5e9759b4fb57c506b633405d4eed417
Author: Johannes Rudolph <johannes_rudolph@gmx.de>
Date:   2010-05-29 16:11:11 +0200

update to scala 2.8.0.RC3


---

diff --git a/objectformatter/pom.xml b/objectformatter/pom.xml
index 634b975..2fe8f30 100644
--- a/objectformatter/pom.xml
+++ b/objectformatter/pom.xml
@@ -5,7 +5,7 @@
   <version>1.0-SNAPSHOT</version>
   <inceptionYear>2008</inceptionYear>
   <properties>
-    <scala.version>2.8.0.Beta1-RC1</scala.version>
+    <scala.version>2.8.0.RC3</scala.version>
   </properties>
 
   <repositories>
diff --git a/objectformatter/with-plugin.sh b/objectformatter/with-plugin.sh
index f0e496a..1ea09c0 100755
--- a/objectformatter/with-plugin.sh
+++ b/objectformatter/with-plugin.sh
@@ -1 +1 @@
-~/downloads/sources/scala-2.8.0.r19928-b20091129020233/bin/scala -Xplugin:target/objectformatter-1.0-SNAPSHOT.jar
+~/downloads/sources/scala-2.8.0.RC3/bin/scala -Xplugin:target/objectformatter-1.0-SNAPSHOT.jar
